### ISSUE TITLE
Streaming over 1080p resolution

### ISSUE THREAD

**Issue by sneaky4oe** [sergey-dryabzhinsky/nginx-rtmp-module#265]
So, I want to stream with 2560x1440p resolution with OBS and have the stream transcoded to 1920x180 with ffmpeg, but when I stream at 1440p, I get a lot of disconnections. I've reported this bug to default RTMP module branch, but seems it's not developed anymore, so I ask here. 

Can you please look into it and help me? I can provide any logs necessary if someone will actually react.

**Comment by UspenskyRuslan** [comment]
Hi there sneaky4oe 

Did you resolved your issue?

I have abput hte same one.

**Comment by sneaky4oe** [comment]
No. The only way of using nginx+rtmp was to rescale output from stream encoder, and keeping the recordings encoder on original scale.
I also stream with AMD rx460 encoder instead of x264 or nvenc - it behaves more stable this way.

**Comment by UspenskyRuslan** [comment]
У меня рад проблем с работой live трансляции

Связка:
OBS + nginx-rtmp-module/mpeg-dash + dash.js
= рандомные лаги на клиенте вида:
getNextFragment - audio - Playing at the bleeding live edge and frag is not available yet

Связка:
OBS + nginx-rtmp-module/hls +hls.js
= рандомные лаги

Связка:
OBS + nginx-rtmp-module + vls плеер
= вообще не заработало (бесконечный прелоадер в плеере)

Итого из 3 связок ни одна не заработала нормально.

Может проблема именно в этом форке?

Завтра попробую пересобрать nginx с оригинальным модулем.

**Comment by UspenskyRuslan** [comment]
OBS показывает что все норм потери покетов нету.

Каналы широкие везде, так что дело не в этом
